### PR TITLE
Update Lancium.yaml

### PR DESCRIPTION
**Pull request by cushingw**
Added squid service entry for Lancium Compute

### CODE REVIEW

**Comment by brianhlin** [comment]
Hi @cushingw, thanks for the PR! For the squid registration, could you register it as a separate resource and leave the backfill registration as is?

See https://github.com/opensciencegrid/topology/blob/85fbd0f0f60a1cc39076db5fecf7fad29a91f157/topology/University%20of%20Wisconsin/CHTC/CHTC.yaml#L294-L310 for an example. Don't worry too much about getting all the fields right, I can drop in suggestions as necessary

**Review by DrDaveD — DISMISSED** [review]

**Comment by DrDaveD** [comment]
@brianhlin it doesn't need to be separate, does it?

**Comment by brianhlin** [comment]
Not strictly but I'd really prefer it if it were: the semantics are fuzzy here with the backfill containers but they're certainly not on the same host as the Squid service

**Comment by DrDaveD** [comment]
I guess they're both behind the same NAT, but not exactly the same resource.  I agree that the resource name doesn't really make sense for the squid.

**Comment by cushingw** [comment]
Is that better?

**Review by DrDaveD — DISMISSED** [review]
The CI check failed because there is no value for the resource "ID:".  A new value can be found by rebasing on the most recent master and  running bin/next_resource_id.  Currently it is 1270.

File: topology/Lancium/Lancium/Lancium.yaml [hunk 1]
```diff
@@ -25,3 +25,24 @@ Resources:
     Services:
       Execution Endpoint:
         Description: Lancium backfill containers
+  Lancium-LMF-Squid:
+    Active: true
+    Description: Squid service located at Lancium LMF datacenter
+    ID:
```
Comment (brianhlin) [inline topology/Lancium/Lancium/Lancium.yaml:31]:
```suggestion
    ID: 1270
```

**Pull request merged**